app/pods/components/tab/template.hbs

```diff
@@ -0,0 +1,9 @@
+<li class="vlc-tabs__tab"...attributes>
+  <LinkTo @route="{{@route}}">
```

Comment:
Moet dit niet zonder de "?  @route=@route ?

Reply:
@route={{@route}} is het enige wat nog gaat